static/app/views/organizationLayout/index.tsx

```diff
@@ -33,20 +37,57 @@ function OrganizationLayout({children}: Props) {
   // oganization is loaded before rendering children. Organization may not be
   // loaded yet when this first renders.
   const organization = useOrganization({allowNull: true});
+  const hasNavigationV2 = organization?.features.includes('navigation-sidebar-v2');
+  const App = hasNavigationV2 ? AppLayout : LegacyAppLayout;
 
   return (
     <SentryDocumentTitle noSuffix title={organization?.name ?? 'Sentry'}>
       <OrganizationContainer>
-        <div className="app">
-          {organization && <OrganizationHeader organization={organization} />}
-          {organization && <DevToolInit />}
-          <Sidebar />
-          <Body>{children}</Body>
-          <Footer />
-        </div>
+        <App organization={organization}>{children}</App>
       </OrganizationContainer>
     </SentryDocumentTitle>
   );
 }
 
+interface LayoutProps extends Props {
+  organization: Organization | null;
+}
+
+function AppLayout({children, organization}: LayoutProps) {
+  return (
+    <AppContainer className="app">
+      <Nav />
+      <BodyContainer>
+        {organization && <OrganizationHeader organization={organization} />}
+        {organization && <DevToolInit />}
+        <Body>{children}</Body>
+        <Footer />
+      </BodyContainer>
+    </AppContainer>
```

Comment:
Notable: adding a new wrapper for the main content allows us to remove the `position: fixed` CSS for the sidebar. This means that we no longer have to hardcode sidebar width and manually adjust padding for the content offset.